.github/workflows/ci.yml

```diff
@@ -36,17 +38,15 @@ jobs:
 
       - uses: actions/checkout@v3
 
-      - name: Get MSRV for ${{ matrix.crate }}
-        id: parse-msrv
+      - name: Get MSRV for $CRATE
         run: |
-          RUST_VERSION=$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[] | select(.name == "${{ matrix.crate }}") | .rust_version')
-          echo "version=${RUST_VERSION}" >> $GITHUB_OUTPUT
-        shell: bash
+          RUST_VERSION=$(cargo metadata --format-version=1 --no-deps | jq -r '.packages[] | select(.name == env.CRATE) | .rust_version')
+          echo "RUST_VERSION=${RUST_VERSION}" >> $GITHUB_ENV
```

Comment:
I think this should probably be renamed to `MSRV` because we compile with latest stable and MSRV.